client/test/e2e/shared/components.js

```diff
@@ -274,5 +283,35 @@ exports.findDebtorGroup = {
     this.popup(),
     this.reload()
   }
-
 };
+
+
+/**
+* Date editor component interface for e2e test
+* @public
+*/
+
+exports.dateEditor = {
+  selector : ['data-date-editor-input'],
```

Comment:
Are you sure you don't mean `'[data-date-editor-input]'` instead of an array of one string?  Does this actually work?